media_driver/linux/common/ddi/media_libva.cpp

```diff
@@ -2279,20 +2294,20 @@ DdiMedia_CreateSurfaces2(
             expected_fourcc = VA_FOURCC_Y216;
             break;
         case VA_RT_FORMAT_YUV444:
-            expected_fourcc = VA_FOURCC_444P;
+            expected_fourcc = VA_FOURCC_XYUV;
```

Comment:
What if the decoder wants a VA_FOURCC_444P? 
IIRC, JPEG decoding for 444P seems to require VA_FOURCC_444P. (I could be wrong since it's been some time since my last check)

Reply:
Thanks linjie's input.
I will use VA_FOURCC_444P as default for yuv444. 
If you want to xyuv, you can use VA_FOURCC_XYUV directly.